app/display.c

```diff
@@ -126,9 +138,9 @@ void display_init(void)
         display_cpu_clk((int)(clks_per_msec / 1000));
     }
     if (cpuid_info.flags.lm) {
-        display_cpu_addr_mode("(x64)");
+        display_cpu_addr_mode("X64");
```

Comment:
Would it be better to use "LM " here?

Reply:
LM is more accurate but probably harder to understand the meaning. Do we need to display the paging mode on 64-bit binary? It's obvisouly LM (well, until we add support for PML5 in a far future...).

Reply:
No, there's currently no need to display it on the 64-bit binary.